submissions/forms.py

```diff
@@ -0,0 +1,30 @@
+  
+from django import forms
+from hackathon.models import HackProject
+
+class SubmissionForm(forms.ModelForm):
+    class Meta:
+        model = HackProject
+        exclude = ('mentor',)
+        fields = ('display_name', 'description','github_url',
+            'deployed_url', 'share_permission', 'speaker_name',)
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        placeholders = {
+            'display_name': 'Project Name',
+            'speaker_name': "Speaker(s) Name",

```

Comment:
Odd combination of 'single' quotes, and "double" quotes.
Let's try to keep it consistent, the majority of the app so far has "double" I think.

Also, I think "Speaker" singular is fine, we only ask that one person speaks during the event.

Reply:
I call it "inconsistency'